source/operations/monitoring/collect-minio-metrics-using-prometheus.rst

```diff
@@ -113,10 +125,8 @@ Append the desired ``scrape_configs`` job generated in the previous step to the
 
 .. tab-set::
 
-   .. tab-item:: Cluster metrics
+   .. tab-item:: Cluster
```

Comment:
Also what to say about `Node`? There's no tab for its scrape config because (I think) it gets included with one of the others? Should clarify that, since it's mentioned above as an option.